doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -54,7 +120,7 @@ Figure 1-1 below shows this at a high level.
 <p align="center"><img src="../figures/ch01_k8s_arch.png" alt="Kubernetes Architecture for NFV" title="Kubernetes Architecture for NFV" width="100%"/></p>
 <p align="center"><b>Figure 1-1:</b> Kubernetes Architecture for NFV</p>
 
-### Other thoughts
+### More thoughts
 - Kubernetes / CaaS should be recognised in ETSI NFV / MANO
 - It is feasible that Kubernetes, using operators and custom resources, could become a generic VNFM (i.e. the Kubernetes parts are the underlying engine, with the operators and custom resources being the equivalent of the specific VNFM).
```

Comment:
A generic VNFM which is specific to the VIM is not really generic...

Reply:
How would it be specific to the VIM?  Not sure I understand?

Reply:
Kubernetes operators will work only on Kubernetes, while in ideal case a VNFM supports OpenStack, Kubernetes and maybe even vCloud...

Reply:
Ah I see - my thoughts were that those VNFM operators could make use of the Cluster API providers to talk to multiple VIMs.  I don't see Kubernetes as a VIM.

Reply:
In my head it is more a VIM/NFVI than any other thing in the ETSI NFV MANO stack, however I agree, that it implements some VNFM features and lacks some VIM/NFVI features. At the end Kubernetes is the component which orchestrates the containers, which are the pieces of infrastructure resources the applications are requesting.

Reply:
Interesting, I would have put the container as part of the application, rather than an infrastructure resource, but I see where you're coming from.

Reply:
When we say generic VNFM, does it mean something like [this](https://openbaton.github.io/documentation/vnfm-generic/) running over kubernetes?

Reply:
K8s sits _on top of_ the NFVI/VIM, because it consumes infrastructure (machines, networks, disks).

It _does_ provide VNFM functionality. And, yes, k8s operators are k8s-specific. At the end of the day, NS-level service+resource orchestrators will have to deal with multiple deployment targets, whether application level (k8s, swarm, ...) or infra level (OpenStack, GCE, ...), via adapters. A common abstraction layer across all these won't fly, just like it didn't above the NFVI/VIM layer.

Reply:
Given this passage in the test has been removed, I'll resolve this conversation.